index.html

```diff
@@ -0,0 +1,111 @@
+ <DOCKTYPE html>
+ <html>
+    <head>
+        <meta charset="utf-8">
+        <title> Барбершоп "Бородинский"</title>
+    </head>
+    <body>
+        <header class="main-header">
+            <nav class="main-navigation">
+                    <a class="main-header-logo">
+                        <img src="img/index-logo.svg" width="168" height="153" alt="Логотип барбершопа Бородинский">
+                    </a>
+                    <ul class="site-navigation">
+                        <li><a href="info.html">Информация</a></li>
+                        <li><a href="news.html">Новости</a></li>
+                        <li><a href="price.html">Прайс-лист</a></li>
+                        <li><a href="shop.html">Магазин</a></li>
+                        <li><a href="contacts.html">Контакты</a></li>
```

Comment:
```suggestion
                        <li><a href="#">Контакты</a></li>
```
Здесь и везде поставь плейсхолдеры, иначе будет по клику 404